spec/mac_app_store_dumper_spec.rb

```diff
@@ -59,6 +59,14 @@
         409201541 Pages (7.1)
         123456789 123AppNameWithNumbers (1.0)
         409203825 Numbers (5.1)
+        944924917 Pastebin It! (1.0)
+        123456789 My (cool) app (1.0)
+        987654321 an-app-i-use (2.1)
+        123457867 App name with many spaces (1.0)
+        893489734 my,comma,app (2.2)
```

Comment:
These are just some (far-fetched) examples of what we should be expecting to accept. Do let me know if you can think of any others we'd like to add here.

Reply:
I think I read in the conversation  on #359 that there is a possibility for emoji in a name. If that's true, it'd be nice to have a test or two for that, with different ranges of emoji characters and maybe non-Latin characters, too..

Reply:
Thanks @colindean! I've pushed up a change includes an emoji and some non-latin characters. Let the encoding issues begin 😛 !